docs/developer/extending/plugins.mdx

```diff
@@ -33,12 +33,13 @@ The plugin configuration can be further validated by overwriting the `validate_p
 # custom_tax/plugin.py
 
 from django.core.exceptions import ValidationError
-from saleor.extensions import ConfigurationTypeField
-from saleor.extensions.base_plugin import BasePlugin
+from saleor.plugins.base_plugin import BasePlugin, ConfigurationTypeField
 
 
 class TaxApiPlugin(BasePlugin):
-    PLUGIN_NAME = "Tax API"
+
+    PLUGIN_ID = "plugin.taxapi"  # plugin identifier
+    PLUGIN_NAME = "Tax API"  # display name of plugin
```

Comment:
Can you also update the structure of the plugin? It was changed a couple of weeks ago and docs seem not up to date